mdr/mdr.py

```diff
@@ -110,7 +143,11 @@ def score(self, features, classes):
             The estimated accuracy based on the constructed feature
 
         """
-        return 0.0
+        new_feature = self.fit_transform(features, classes)
```

Comment:
This line should only `transform`, not `fit`. The MDR model should already be fitted at this point.

That said, we should have a check here to make sure that `fit` has been called already and raise a `ValueError` if not.